src/main/java/io/specto/hoverfly/junit/core/Hoverfly.java

```diff
@@ -265,6 +281,13 @@ public void simulate(SimulationSource simulationSource, SimulationSource... sour
                         simulation.getHoverflyData().getGlobalActions().getDelays().addAll(s.getHoverflyData().getGlobalActions().getDelays());
                     });
 
+            if (isSimulationPreprocessingEnabled(testInstance)) {
```

Comment:
This method `isSimulationPreprocessingEnabled(testInstance)` is called twice, can we set it as a variable?